sqlair.go

```diff
@@ -81,14 +76,12 @@ func (db *DB) PlainDB() *sql.DB {
 
 // Query holds the results of a database query.
 type Query struct {
-	pq      *expr.PrimedQuery
-	sqlstmt *sql.Stmt
-	ctx     context.Context
-	err     error
-	tx      *TX // tx is only set for queries in transactions.
-	// Persist statement and db so the sql.Stmt is not closed until the Query is dropped.
-	stmt *Statement
-	db   *DB
+	// run executes the Query against the db or the tx.
+	run func() (*sql.Rows, sql.Result, error)
+	ctx context.Context
+	err error
+	pq  *expr.PrimedQuery
+	db  *DB
```

Comment:
However, this `db` is now unused, so we can remove it.

Reply:
You are right! These were in part used to prevent running the finalizers which is no longer happening.